guillotina/json/serialize_content.py

```diff
@@ -155,7 +155,7 @@ def check_permission(self, permission_name):
             if permission is None:
                 self.permission_cache[permission_name] = True
             else:
-                security = IInteraction(self.request)
+                security = get_utility(ISecurityPolicy)
```

Comment:
Security as utility is shared across records?

Reply:
switched to adapter.